cmd/otel-allocator/README.md

```diff
@@ -1,14 +1,25 @@
 # Target Allocator
 
-The TargetAllocator is an optional separately deployed component of an OpenTelemetry Collector setup, which is used to
-distribute targets of the PrometheusReceiver on all deployed Collector instances. The release version matches the
+Target Allocator is an optional component of the OpenTelemetry Collector [Custom Resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (CR). The release version matches the
 operator's most recent release as well.
 
-In essence, Prometheus Receiver configs are overridden with a http_sd_configs directive that points to the
-Allocator, these are then loadbalanced/sharded to the collectors. The Prometheus Receiver configs that are overridden
-are what will be distributed with the same name. In addition to picking up receiver configs, the TargetAllocator
-can discover targets via Prometheus CRs (currently ServiceMonitor, PodMonitor) which it presents as scrape configs
-and jobs on the `/scrape_configs` and `/jobs` endpoints respectively.
+In a nutshell, the TA is used to manage Prometheus metrics without actually needing to have Prometheus installed in your Kubernetes cluster.
+
+The TA serves two functions:
+* Even distribution of Prometheus targets among a pool of Collectors
+* Discovery of Prometheus Custom Resources
+
+## Even Distribution of Prometheus Targets
+
+The Target Allocator's first job is to evenly distribute [Prometheus targets](https://www.tigera.io/learn/guides/prometheus-monitoring/prometheus-metrics/#:~:text=Prometheus%20monitoring%20works%20by%20identifying,and%20generates%20metrics%20from%20it.) among a pool of OTel Collectors. This means that the OTel Collectors collect the metrics instead of a Prometheus [scraper](https://uzxmx.github.io/prometheus-scrape-internals.html). Metrics are ingested by the OTel Collectors by way of the [Prometheus Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/prometheusreceiver/README.md).
```

Comment:
The first job is to discover both targets to scrape and collectors to allocate targets to.  Then it can distribute the targets it discovers among the collectors.

https://uzxmx.github.io/prometheus-scrape-internals.html is probably not the best link for this statement as the OTel collector actually uses all of those scraper internals in its prometheus receiver.